src/helpers/validators.js

```diff
@@ -0,0 +1,78 @@
+export const validateHelper = {
+  isEmail: value => {
+    const res = value
+      .toString()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      );
+    return res !== null;
+  },
+  isUrl: value => {
+    const res = value
+      .toString()
+      .match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
+    return res !== null;
+  },
+  isNumber: value => {
+    const res = value.toString().match(/\D/);
+    return res === null;
+  },
+  isEmpty: value => value.toString().trim() === "",
+  isLength: (value, len) => !!(value && value.toString().length === len),
+  isMinLength: (value, len) => !!(value && value.toString().length >= len),
+  isMaxLength: (value, len) => !!(value && value.toString().length <= len),
+  isAlpha: value => {
+    const res = value.toString().match(/^\W/);
```

Comment:
Regex to be constant. Safe navigation operator for value

Reply:
updated with 
`isAlphaNumeric: value => {
    const res = value?.toString().match(NON_ALPHA_NUMERIC_PATTERN);
    return typeof res !== "undefined" && res == null;
  },`